storage/conformance_test.go

```diff
@@ -172,7 +174,24 @@ func TestSigningV4Conformance(t *testing.T) {
 					style = BucketBoundHostname(tc.BucketBoundHostname)
 				}
 
-				gotURL, err := SignedURL(tc.Bucket, tc.Object, &SignedURLOptions{
```

Comment:
Out of curiosity: are other languages which don't have an equivalent of our `BucketHandle.SignedURL` able to handle this correctly? Do they need to add more signed URL options so the user can update the configuration manually?

Reply:
For clients that don't already support it, the guidance is for languages to add a method somewhere below the client level to handle this (and eventually provide users with client creds re-use).